core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -76,19 +80,50 @@ class MultiCloudWorkspaceService(userInfo: UserInfo,
       case Some(value) => value
     }
 
-    // for now, the only supported azure billing project is the hardcoded one from the config
+    // Temporary hard-coded Azure billing account, to be removed when users can create Azure-backed billing projects in Terra.
     if (workspaceRequest.namespace == azureConfig.billingProjectName) {
+      val azureConfig = multiCloudWorkspaceConfig.azureConfig.getOrElse(throw new RawlsException("Azure config not present"))
       createMultiCloudWorkspace(
         MultiCloudWorkspaceRequest(
           workspaceRequest.namespace,
           workspaceRequest.name,
           workspaceRequest.attributes,
           WorkspaceCloudPlatform.Azure,
-          azureConfig.defaultRegion
+          azureConfig.defaultRegion,
+          AzureManagedAppCoordinates(
+            UUID.fromString(azureConfig.azureTenantId),
+            UUID.fromString(azureConfig.azureSubscriptionId),
+            azureConfig.azureResourceGroupId
+          ),
+          azureConfig.spendProfileId
         )
       )
     } else {
-      workspaceService.createWorkspace(workspaceRequest, parentSpan)
+      traceWithParent("withBillingProjectContext", parentSpan)(
```

Comment:
@aherbst-broad switched to using ` workspaceService.withBillingProjectContext` and passed in `parentSpan`. Please check that I did the span stuff correctly.